Assets/VRTK/Scripts/Interactions/VRTK_InteractGrab.cs

```diff
@@ -100,9 +104,30 @@ public GameObject GetGrabbedObject()
 
         protected virtual void Awake()
         {
-            interactTouch = GetComponent<VRTK_InteractTouch>();
-            controllerActions = GetComponent<VRTK_ControllerActions>();
-            controllerEvents = GetComponent<VRTK_ControllerEvents>();
+            if (controllerEvents == null)
+            {
+                controllerEvents = GetComponent<VRTK_ControllerEvents>();
+            }
+            if (controllerEvents == null)
+            {
+                throw new System.Exception("InteractGrab requires a non-null VRTK_ControllerEvents instance. The field controllerEvents isn't set and there was no instance found on gameObject.");
```

Comment:
Do we want to throw exceptions or do `Debug.LogError` which is used in other places to report an issue